service-front/module/Application/config/module.config.php

```diff
@@ -83,13 +103,53 @@
             'passport_number' => [
                 'type'    => Segment::class,
                 'options' => [
-                    'route'    => '/passport-number',
+                    'route'    => '[/:uuid]/passport-number',
                     'defaults' => [
                         'controller' => Controller\IndexController::class,
                         'action'     => 'passportNumber',
                     ],
                 ],
             ],
+            'id_verify_questions' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '[/:uuid]/id-verify-questions',
+                    'defaults' => [
+                        'controller' => Controller\IndexController::class,
```

Comment:
Similarly here, might want to think about breaking these up into multiple controllers rather than maintaining one massive file